### DISCUSSION TITLE
Add "Filter by Field" to Table Layout Column Header Menu

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#16702] [Feature Requests]
![CleanShot 2022-12-02 at 12 30 14](https://user-images.githubusercontent.com/522079/205351027-b50774d2-3bd3-4d5a-9de3-562e951fe29e.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
